### PR TITLE
css-positioning -Daniel Cho

### PR DESCRIPTION
**Pull request by danielycho**
relative
absolute
Fixed

### CODE REVIEW

File: css-positioning/absolute.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+  position: absolute;
+  left: 225px;
+  bottom: 225px;
```
Comment (codingcodymiller) [inline css-positioning/absolute.css:19]: Oh no, this is a _**big**_ movement. In general, you should try to use the box offset properties closest to the edge you want to position this element at. In this case, that is `top` and `right`

File: css-positioning/absolute.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+  position: absolute;
+  top: 165px;
```
Comment (codingcodymiller) [inline css-positioning/absolute.css:33]: Same here, the closer side would be `bottom`

File: css-positioning/absolute.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+  /* your code here */
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgb(0 0 0 / 75%));
+
```
Comment (codingcodymiller) [inline css-positioning/absolute.css:40]: You missed this :eyes:

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-positioning/absolute.css (hunk 3 above)
Comment (danielycho) [reply css-positioning/absolute.css:45]: image

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**